nixos/modules/services/web-apps/jitsi-meet.nix

```diff
@@ -0,0 +1,503 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.jitsi-meet;
+  attrsToArgs = a: concatStringsSep " " (mapAttrsToList (k: v: "${k}=${toString v}") a);
+
+  # The configuration files are JS of format "var <<string>> = <<JSON>>;". In order to
+  # override only some settings, we need to extract the JSON, use jq to merge it with
+  # the config provided by user, and then reconstruct the file.
```

Comment:
Has there any discussion with upstream about why this is javascript, and not a "proper" configuration file format, like json, yaml or toml, or HOCON - which they use in other parts?

Reply:
I opened an issue about this: jitsi/jitsi-meet#5590 with no response so far.